### DISCUSSION TITLE
"error connecting to ledger device" in Firefly

### DISCUSSION THREAD

**Discussion by mllechelle** [iotaledger/firefly#7663] [Help & Support]
Hi all,

I am able to connect my nano x to firefly (at least firefly say the ledger device is unlocked and open) but when I want to collect funds from vesting or manage my voting power I am told "there was an error connecting to your ledger device". Any idea what is going on?
Really grateful for any help.
Cheers

**Comment by begonaalvarezd** [comment]
You need the latest ledger app version, 0.8.6, if you cant see it in ledger live app, you will need to upgrade your firmware
Please let us know if the issue resolves, thanks 🌷

**Reply by mllechelle** [reply]
Thank you - I am discussing with ledger why I cant update the device in the ledger live app.

**Reply by begonaalvarezd** [reply]
Sorry, I realized i may have explained myself wrong, what I meant is that you need is the **iota ledger app** version **0.8.6** 🙏🏼 If you cant see the version in ledger live, you need to upgrade your firmware, and if you cant even upgrade your firmware, you might need to start by updating your ledger live to latest

**Reply by mllechelle** [reply]
Thank you very much for your quick response. I fixed the issue and now all is fine.
Appreciate the help!
